wagtail/images/models.py

```diff
@@ -450,7 +450,7 @@ def filter_operations(self):
             if isinstance(operation, FilterOperation)
         ]
 
-    def get_transform(self, image):
+    def get_transform(self, image, transform_width=None, transform_height=None):
```

Comment:
Just for consistency, shall we pass the width and height together as a tuple?

Reply:
> Just for consistency, shall we pass the width and height together as a tuple?

Makes sense to me. I asked about that up in the main comments. I wasn't sure if a tuple or named arguments would be preferred since the named parameters make it extra explicit what those are, but do leave more room for mistakes as well.

I'd be happy to make it be a tuple(width, height) and then just document that in the docstring. I should double check for other docs which need updated as well. I wanted to get this draft up and make sure this was an ok approach before changing much else. I'll try to get that and tests tonight - woorst case it'll be tomorrow night.

Reply:
@kaedroho I've added tests for the Rendition model. I borrowed the tests from Willow which you pointed me at and tweaked them to check the initial dimensions, then generate a rendition, and then check the dimensions of the rendition model. It also goes through Willow to open the files so that it can verify the actual image file's dimensions are still correct and that the correct colored pixels are where they belong.